src/datasources/workspace/WorkspaceDataSource.test.ts

```diff
@@ -4,17 +4,23 @@ import _ from 'lodash';
 import {
   createFetchError,
   createFetchResponse,
+  getQueryBuilder,
+  mockTimers,
   requestMatching,
   setupDataSource,
 } from 'test/fixtures';
 import { WorkspaceDataSource } from './WorkspaceDataSource';
+import { WorkspaceQuery } from './types';
 
 let ds: WorkspaceDataSource, backendSrv: MockProxy<BackendSrv>;
 
 beforeEach(() => {
   [ds, backendSrv] = setupDataSource(WorkspaceDataSource);
 });
 
+const buildQuery = getQueryBuilder<WorkspaceQuery>()({});
+mockTimers();
```

Comment:
I don't think this is necessary since `setTimeout()` won't be called from these tests. This was needed in the tag tests to be able to test the retry logic.